attelo/decoding/mst.py

```diff
@@ -1,88 +1,116 @@
 '''
 Created on Jun 27, 2012
 
-@author: stergos
-
-
+@author: stergos, jrmyp
 '''
 
-from collections import defaultdict
-from math import log
+from __future__ import print_function
 import sys
 
+from collections import defaultdict
+from scipy.special import logit
+
 from depparse.graph import Digraph
 from .interface import Decoder
 
 # pylint: disable=too-few-public-methods
 
+# Required to work with depparse, working with a "minimum" score of 1e-100
+# Unless we have more than 1e10 nodes, combined scores shouldn't reach it
+MAX_SCORE = 1e90
```

Comment:
(is it -1e100 or 1e-100?) 

Maybe this should be a bit more explicit, for example something like “Required because the depparse package internally uses -1e100 to represent its minimum score.  Going below this 'minimum' *crashes* depparse”